src/main/java/org/jabref/logic/journals/LwtaAbbreviationRepository.java

```diff
@@ -0,0 +1,209 @@
+package org.jabref.logic.journals;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class LwtaAbbreviationRepository {
+
+    private final Map<String, LwtaAbbreviation> lwtaToAbbreviationObject;
+
+    // incomplete list
+    private final String[] WORDS_TO_REMOVE = new String[]{"the", "and", "&", "of", "but", "sans", "section", "series", "part"};
+
+    /**
+     * instantiates this class with a csv file
+     */
```

Comment:
Remove comment - or replace it by an explenation of the `@param file`  parameter